sdcm/nemesis.py

```diff
@@ -253,6 +278,7 @@ def __init__(self, tester_obj, termination_event, *args, nemesis_selector=None,
         }
         self.es_publisher = NemesisElasticSearchPublisher(self.tester)
         self._init_num_deletions_factor()
+        self.set_target_node_pool(self.cluster.data_nodes)
```

Comment:
more clear would be to explicitly show protected attribute `_target_node_pool`:
`self._target_node_pool = self.cluster.data_nodes`